kim_tools/ase/core.py

```
@@ -933,262 +937,1284 @@ def energy_cheat(a, dimer, large_cell_len, offset, einf):
+            nelder_mead_timeout=nelder_mead_timeout,
+            max_starting_points=max_starting_points,
+            ncells_per_side=1,
+            seed=seed,
+            coarse_a_start=coarse_a_start,
+            coarse_a_stop=coarse_a_stop,
+        )
+
+        mixed_result["approx_mixed_equilibrium_alat"] = approx_mixed_equilibrium_alat
+
+        final_result = mixed_result
+        results = mono_results + [mixed_result]
```

Ruff (0.16.0)

[warning] Consider `[*mono_results, mixed_result]` instead of concatenation

Replace with `[*mono_results, mixed_result]`

(RUF005)